src/main/java/nl/tudelft/ti2206/group9/gui/ExternalTicker.java

```diff
@@ -71,11 +71,11 @@ public ExternalTicker() {
     @Override
     public final void handle(final long now) {
         synchronized (ShaftEscape.TICKER_LOCK) {
-            if (InternalTicker.isRunning()) {
-                GameScene.getAudioPlayer().setSpeed(1
-                        + (Track.getUnitsPerTick()) / RATE_DIVISION);
-                GameScene.getAudioPlayer().play(true);
-            }
+// Controls the discrete increase of soundtrack, temporarily disabled!
+//            if (InternalTicker.isRunning()) {
+//                GameScene.getAudioPlayer().setSpeed(1
+//                        + (Track.getUnitsPerTick()) / RATE_DIVISION);
+//            }
```

Comment:
Once you add continuous increase of speed, I would like to see an `SoundtrackPlayer.update()` method of some kind, the ExternalTicker should not know _how_ to increase the soundtrack, but only has to call the one responsible.

<review,interface,orgainization